### PR TITLE
intro-screen

### PR DESCRIPTION
**Pull request by hnllong**
#### What changed:
-Intro screen
-Login Screen
-Register Screen
#### Please check if the PR fulfills these requirements:
- [x] The commit message follows our guidelines
- [ ] Passed on Android
- [x] Passed on iOS
#### Free comment:
- N/A
#### Screenshoot (if task changed UI)
simulator_screenshot_5E5387F4-FD56-4658-81FC-A3F04E8F71E9
simulator_screenshot_1DACBE4B-F3D0-4ACF-8225-C36EFC8F3068
simulator_screenshot_D49F03EC-27BA-4844-AEF6-DFB24E9418C2

### CODE REVIEW

File: src/feature/authentication/RegisterTrainingScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    );
+};
+
+const styles = ScaledSheet.create({
+    container: {
+        flex: 1,
+    },
+    main: {
+        flex: 1,
+        backgroundColor: 'black',
```
Comment (duongquang1611) [inline src/feature/authentication/RegisterTrainingScreen.tsx]: màu import từ file themes nhá

File: src/feature/authentication/RegisterTrainingScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    titleRegister: {
+        color: '#fff',
```
Comment (duongquang1611) [inline src/feature/authentication/RegisterTrainingScreen.tsx]: Màu này cũng thế, check cả pull xem còn chỗ nào không nhé

File: src/feature/intro/IntroScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+const styles = ScaledSheet.create({
+    container: {
+        flex: 1,
+    },
+    introBg: {
+        width: '100%',
+        height: '100%',
+    },
+    txtIntro: {
+        height: 89,
```
Comment (duongquang1611) [inline src/feature/intro/IntroScreen.tsx]: Check cả pull request này rồi sửa dùng ScaledSheet nhá

Comment (hnllong) [reply src/feature/intro/IntroScreen.tsx]: Done

**Pull request merged**